packages/smooth_app/lib/pages/preferences/user_preferences_dev_mode.dart

```diff
@@ -390,6 +401,47 @@ class UserPreferencesDevMode extends AbstractUserPreferences {
     );
   }
 
+  Future<void> _changePrimaryColor(
+    BuildContext context,
+    UserPreferences userPreferences,
+  ) async {
+    Color? newColor;
+
+    void changeColor(Color color) {
+      newColor = color;
+    }
+
+    final ThemeProvider themeProvider = context.read<ThemeProvider>();
+
+    final bool? apply = await showDialog<bool>(
+      context: context,
+      builder: (BuildContext context) {
+        return SmoothAlertDialog(
+          title: 'Pick a color!',
+          body: SingleChildScrollView(
+            child: ColorPicker(
+              pickerColor: themeProvider.color,
+              onColorChanged: changeColor,
+            ),
+          ),
+          actions: <SmoothActionButton>[
+            SmoothActionButton(
+              text: 'Got it',
+              onPressed: () {
+                Navigator.of(context).pop(true);
+              },
+            ),
+          ],
+        );
+      },
+    );
+
+    if (apply == null || newColor == null) {
```

Comment:
You could maybe just say `if (values are true) -> themeProvider.setColor...`
I don't think the return alone is really useful here